docs/sources/mimir/configure/tuning.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Learn how to tune Grafan Mimir according to your use cases.
+menuTitle: Tuning
+title: Tune Grafana Mimir according to your use cases
+weight: 110
+---
+
+# Tune Grafana Mimir according to your use cases
+
+Grafana Mimir comes with sensible default settings. Those settings are a good place to start for most use cases.
+However, for some use cases Grafana Mimir requires appropriate tuning to reach optimal performance. This page aims to centralize those known tuning.
+
+## Heavy multi-tenancy
+
+For each tenant, Grafana Mimir opens and maintains a TSDB in memory. With a significant number of tenants the memory overhead might come prohibitive.
+To reduce the associated overhead, users might consider:
+
+- Reduce `-blocks-storage.tsdb.head-chunks-write-buffer-size-bytes`, default `4MB`. For example try `1MB`, or `128KB`.
+- Reduce `-blocks-storage.tsdb.stripe-size`, default `16384`. For example try `256` or even `64`.
+- Configure [shuffle sharding]({{< relref "./configure-shuffle-sharding" >}})
+
+## Compression
+
+Depending on the CPU model used in the underlying infrastructure, the compression for both WALs and GRPC communication might consumes a significant part of the available CPU resources.
+To identify such case one could rely on profiling with tools like [Grafana Pyroscope](https://grafana.com/docs/pyroscope/latest/)
+
+To reduce the resource consumption, users might consider:
+
+- Make sure `wal_compression_enabled` is not enable
+- Make sure `grpc_compression` is either off which is the default or configured to `snappy`. `gzip` consumes more CPU than `snappy`. However, disabling `grpc_compression` implies more network traffic and in turn might increase the total cost of ownership (TCO) of running Grafana Mimir.
+
+If users must used compression, like for example to fit in the network bandwidth, they might consider using nodes with more powerful CPU. This implies a increase of the TCO.
+
+## Increase the cache size on a budget
+
+Grafana Mimir relies on Memcached for its caches. Memcached relies, by default only on the memory.
+Similarly to the work of the Ops team behind Grafana Loki, one could enable the `extstore` feature of Memcached.
```

Comment:
I don't think it's necessary to include what the Loki team is doing here. Rather, could we provide a brief description as to what enabling `extstore` accomplishes?